versioned_docs/version-v1.4/rancher/rancher-integration.md

```diff
@@ -152,12 +152,21 @@ The Rancher feature `harvester-baremetal-container-workload` must be enabled.
 
   ![](/img/v1.3/rancher/gitrepo-definition.png)
 
-1. Click **Next**, and then define the Git repo targets. You can select all clusters, an individual cluster, or a group of clusters. 
+1. Click **Next**, and then define the Git repo targets. You can select all clusters, an individual cluster, or a group of clusters.
 
   In this example, the cluster group named `private-dc-clusters` is used.
 
-  ![](/img/v1.3/rancher/gitrepo-targets.png) 
+  ![](/img/v1.3/rancher/gitrepo-targets.png)
 
 1. Click **Save**. It may take a few seconds for the resources to be rolled out to the target clusters.
 
-  ![](/img/v1.3/rancher/gitrepo-synced.png)
\ No newline at end of file
+  ![](/img/v1.3/rancher/gitrepo-synced.png)
+
+## Work with prime Rancher
+
+The prime Rancher uses different registry. If there is no `CATTLE_AGENT_IMAGE` environment variable, the default value uses `docker.io` as default registry whick makes the Harvester can't get correct rancher-agent image. You need to set the `CATTLE_AGENT_IMAGE` environment variable to the correct registry.
```

Comment:
our doc doesn't suggest using the docker way to deploy Rancher, does Helm deployment need this?